drivers/usbhost/Kconfig

```diff
@@ -286,7 +286,7 @@ config HIDKBD_DEFPRIO
 
 config HIDKBD_STACKSIZE
 	int "Polling thread stack size"
-	default 1024
```

Comment:
It is inappropriate to increase the size of this tuned stack size.  Let's not do that.